src/main/java/fr/esgi/al5_2/Tayarim/entities/Administrateur.java

```diff
@@ -0,0 +1,24 @@
+package fr.esgi.al5_2.Tayarim.entities;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.LocalDateTime;
+import java.util.List;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'java.util.List' import. Should be before 'lombok.*'.